speech/grpc/transcribe_streaming.py

```diff
@@ -87,22 +85,28 @@ def _audio_data_generator(buff):
                 data.append(buff.get(block=False))
             except queue.Empty:
                 break
+
+        # If the data contains None then set stop = True.
```

Comment:
Comments should describe the meaning and purpose of the code, not just narrate what the code is doing. Maybe something like:

```
# If `_fill_buffer` adds `None` to the buffer, the audio stream is closed.
# Yield the final bit of the buffer and exit the loop.
```